corehq/ex-submodules/phonelog/tasks.py

```diff
@@ -24,6 +27,11 @@ def purge_old_device_report_entries():
     UserEntry.objects.filter(server_date__lt=max_age).delete()
 
 
-@task(queue='sumologic_logs_queue')
-def send_device_logs_to_sumologic(domain, xform, url):
-    SumoLogicLog(domain, xform).send_data(url)
+@no_result_task(queue='sumologic_logs_queue', default_retry_delay=10 * 60, max_retries=3, bind=True)
+def send_device_log_to_sumologic(self, url, data, headers):
+    with Session() as s:
+        s.mount(url, HTTPAdapter(max_retries=5))
```

Comment:
Does this mean it will try 15 times? 3 celery task, 5 time each?

Reply:
yes